lisp/i18n/ts/fr_FR/lisp.ts

```diff
@@ -1415,17 +1395,17 @@
     <message>
       <location filename="../../../ui/logging/dialog.py" line="79"/>
       <source>Dismiss all</source>
-      <translation type="unfinished">Dismiss all</translation>
+      <translation>Tout ignorer</translation>
     </message>
     <message>
       <location filename="../../../ui/logging/dialog.py" line="86"/>
       <source>Show details</source>
       <translation>Afficher les détails</translation>
     </message>
     <message>
-      <location filename="../../../ui/logging/viewer.py" line="55"/>
+      <location filename="../../../ui/logging/viewer.py" line="53"/>
       <source>Linux Show Player - Log Viewer</source>
-      <translation type="unfinished">Linux Show Player - Log Viewer</translation>
+      <translation>Lecteur Linux Show - Visionneur de log</translation>
```

Comment:
We should keep the name "Linux Audio Player" as a whole, and not translate it.
"log" -> "journal"